packages/platform-sdk-profiles/src/drivers/memory/repositories/wallet-repository.ts

```diff
@@ -234,7 +242,10 @@ export class WalletRepository implements IWalletRepository {
 
 			wallet.settings().fill(settings);
 
-			await wallet.mutator().coin(coin!, network, { sync: false });
+			await wallet.mutator().coin(
+				new CoinFactory(this.#profile).make(coin, network),
```

Comment:
Here we also need a coin instance but we only have the coin and network name at this point.